From c450d2334d59c2441d9b4fcee1284afb55b9639c Mon Sep 17 00:00:00 2001
From: Stijn Tintel <stijn@linux-ipv6.be>
Date: Wed, 1 Sep 2021 03:32:18 +0300
Subject: [PATCH] rasdaemon: fix compile against musl libc

Fix the following compile errors that occurs when building against musl:

ras-events.c: In function 'read_ras_event_all_cpus':
ras-events.c:366:16: error: 'PATH_MAX' undeclared (first use in this function)
  366 |  char pipe_raw[PATH_MAX];
      |                ^~~~~~~~

ras-events.c: In function 'handle_ras_events_cpu':
ras-events.c:564:16: error: 'PATH_MAX' undeclared (first use in this function)
  564 |  char pipe_raw[PATH_MAX];
      |

Signed-off-by: Stijn Tintel <stijn@linux-ipv6.be>
---
 ras-events.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/ras-events.c b/ras-events.c
index fe4bd26..39cab20 100644
--- a/ras-events.c
+++ b/ras-events.c
@@ -18,6 +18,7 @@
 #include <dirent.h>
 #include <errno.h>
 #include <fcntl.h>
+#include <limits.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
